packages/regl-worldview/src/commands/Command.js

```diff
@@ -127,35 +136,102 @@ export default class Command<T> extends React.Component<Props<T>> {
   }
 }
 
-function getHitmapProps(getHitmapId, children) {
-  if (!getHitmapId || !children || children.length === 0) {
+// TODO: deprecating, remove before 1.x release
+function defaultGetHitmapProps<T>(getHitmapId, children: T[]): ?(HitmapProp[]) {
+  if (!children || children.length === 0) {
     return undefined;
   }
 
   return children.reduce((memo, marker) => {
-    const hitmapId = getHitmapId(marker);
-    // filter out components that don't have hitmapIds
-    if (hitmapId != null) {
-      memo.push({
-        ...marker,
-        color: intToRGB(getHitmapId(marker) || 0),
-      });
+    if (marker.color) {
```

Comment:
I don't think the marker having a `.color` property should have bearing on whether or not it's passed to the provided `getHitmapId` function - markers without colors could be a thing in custom `<Command />` components which have colors hard coded in the shaders or use textures or other ways to provide colors to webgl.  I suggest this if check be removed.